pkg/filter/http/grpcproxy/grpc.go

```diff
@@ -189,7 +189,8 @@ func (f *Filter) Decode(c *http.HttpContext) filter.FilterStatus {
 	e := server.GetClusterManager().PickEndpoint(re.Cluster, c)
 	if e == nil {
 		logger.Errorf("%s err {cluster not exists}", loggerHeader)
-		c.SendLocalReply(stdHttp.StatusServiceUnavailable, []byte("cluster not exists"))
+		errResp := http.ServiceUnavailable.WithError(fmt.Errorf("cluster not exists"))
```

Comment:
if the format func is not used when creating a new err, use `errors.New` instead of `fmt.Errorf`. `fmt.Errorf` only used when need format

```suggestion
		errResp := http.ServiceUnavailable.WithError(errors.New("cluster not exists"))
```